packages/components/package.json

```diff
@@ -48,7 +48,9 @@
     "react-dnd-html5-backend": "^2.5.4",
     "react-immutable-proptypes": "^2.1.0",
     "react-virtualized": "9.21.0",
-    "simulant": "^0.2.2"
+    "reactour": "^1.13.1",
+    "simulant": "^0.2.2",
+    "styled-components": "^4.1.3"
```

Comment:
😏

Reply:
I had to, I'm still convinced it's not a good idea :)

Reply:
https://bundlephobia.com/result?p=styled-components@4.1.3

Reply:
Is that bad?

Reply:
this is big

Reply:
Do you prefer that I put it as optional?

Reply:
I prefer to be loaded using lazy / suspense if possible.
So if the user already know the app it s not downloaded.